apps/docs/app/ui/components/previewComponent/PlaygroundWrapper.tsx

```diff
@@ -0,0 +1,25 @@
+"use client";
+
+import clsx from "clsx";
+
+import { SandpackLayout, SandpackPreview, SandpackProvider } from "@codesandbox/sandpack-react";
+
+const PlaygroundWrapper = ({ key, className }: { key: string; className?: string }) => {
```

Comment:
This comment will be repeated a couple of time:

- Create an interface instead of inline types.
- is the key prop really necessary?
- clsx(classname) should be replaced by className={className}

Reply:
ill stop copying the 3 comments now